website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -781,6 +796,8 @@ The `addon_profile` block exports the following:
 
 * `oms_agent` - An `oms_agent` block as defined below.
 
+* `azure_keyvault_secrets_provider` - An `azure_keyvault_secrets_provider` block as defined below.
```

Comment:
Could we move this up before `ingress_application_gateway`? The properties should be alphabetically sorted within blocks